package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@krosas/bimsurfer3",
-    "version": "0.272.1",
+    "version": "0.273.1",
```

Comment:
si nechcel meniť druhé číslo, to je verzia surfer-a z ich číslovania

Reply:
este som si povedal ze ved to je feature.. ok jasne dam to .2